pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -432,7 +432,7 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 				}
 
 				for _, cn := range clusters {
-					s.addEdsCon(cn, con.ConID, con)
+					s.getOrAddEdsCluster(cn, con.ConID, con)
```

Comment:
Is this change a result of a linter complaint?

Reply:
Maybe I went a bit too far with this one. The linter error was `addEdsCon` having a bunch of commented out code. It turned out this unfction now just called `getOrAddEdsCluster` with the same args so I cleaned it up a bit. I can revert this and do the minimal change though if that is easier, since we already have tons of other fixes here